### PR TITLE
Wifi update for setting SSID/Password/Connection Status

### PR DESCRIPTION
**Pull request by sureshven**

### CODE REVIEW

File: MojioSDK/Client/AuthClient.swift [hunk 1]
```diff
@@ -276,7 +276,7 @@ public class AuthClient: NSObject, AuthControllerDelegate {
         // The token endpoint is used for the resource owner flow
         let loginEndpoint = AuthClient.getTokenUrl()
         
-        Alamofire.request(.POST, loginEndpoint, parameters: ["grant_type" : "password", "password" : password, "username" : username, "client_id" : self.clientId, "client_secret" : self.clientSecretKey], encoding: .URL, headers: ["Accept" : "application/json", "Authorization" : self.generateBasicAuthHeader()]).responseJSON{ response in
+        let request = Alamofire.request(.POST, loginEndpoint, parameters: ["grant_type" : "password", "password" : password, "username" : username, "client_id" : self.clientId, "client_secret" : self.clientSecretKey], encoding: .URL, headers: ["Accept" : "application/json", "Authorization" : self.generateBasicAuthHeader()]).responseJSON{ response in
```
Comment (agarwalashish) [inline MojioSDK/Client/AuthClient.swift]: We should not do`let request =  ` if we're not using the value of request. It simply adds to the number of warnings ("unused variables) in the app

File: MojioSDK/Client/AuthClient.swift [hunk 2]
```diff
@@ -306,6 +306,10 @@ public class AuthClient: NSObject, AuthControllerDelegate {
                 failure(response: response.result.value as? NSDictionary)
             }
         }
```
Comment (agarwalashish) [inline MojioSDK/Client/AuthClient.swift]: I would not add print statements in the SDK. 3rd party developers will not want to see such a vast amount of (potentially meaningless) information in their consoles. It's fine for debugging locally, but take it out of the SDK.

File: MojioSDK/Models/WifiRadio.swift [hunk 3]
```diff
@@ -10,18 +10,52 @@ import UIKit
...
+        }
+
+        if let password = self.Password {
+            dictionary.setObject(password, forKey: "Password")
+        }
+        
+        if let status = self.Status {
+            dictionary.setObject(status, forKey: "Status")
+        }
+
```
Comment (agarwalashish) [inline MojioSDK/Models/WifiRadio.swift]: Add a comment somewhere around ttl (Time To Live). A lot of developers will not be familiar with the terminology

Comment (sureshven) [inline MojioSDK/Models/WifiRadio.swift]: ttl is just a temp variable since its testing time to live.

File: MojioSDK/Client/AuthClient.swift (hunk 2 above)
Comment (sureshven) [inline MojioSDK/Client/AuthClient.swift]: Print statements are useful for debugging and are not part of the release build #if/def'd

Comment (agarwalashish) [inline MojioSDK/Client/AuthClient.swift]: Correct, but third party developers will be developing the app in debug mode.

Comment (sureshven) [inline MojioSDK/Client/AuthClient.swift]: You have to be in debug mode and add -DDEBUG to the flags for the print to show up.

Comment (agarwalashish) [inline MojioSDK/Client/AuthClient.swift]: Let's take them out for the sake of simplicity. It makes it hard for 3rd party devs (even for us) when one has to dig into a bunch of settings to see why code is/isn't working.

Comment (sureshven) [inline MojioSDK/Client/AuthClient.swift]: That makes no sense.

File: MojioSDK/Client/AuthClient.swift (hunk 1 above)
Comment (sureshven) [inline MojioSDK/Client/AuthClient.swift]: Hmm - I'm not seeing a warning on a clean project build.

File: MojioSDK/Client/AuthClient.swift (hunk 2 above)
Comment (agarwalashish) [inline MojioSDK/Client/AuthClient.swift]: If I were to look at the codebase with a fresh set of eyes, I would not know what `#if DEBUG ` meant. To a general person, it means that the build should be in debug mode and -DDBUG is not the first thing to cross my mind. It's possible to spend a lot of time trying to figure out why the warnings aren't showing up when in theory they should.

Comment (sureshven) [inline MojioSDK/Client/AuthClient.swift]: No worries - I'm updating the PR.

**Comment by sureshven** [comment]
Updated with comment on time to live, removed auth client changes and updated project so SDK can build correctly from SDK workspace.

File: MojioSDK.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -438,15 +441,15 @@
 			name = Models;
 			sourceTree = "<group>";
 		};
-		E0E63D2D1D21DF3200DDF336 /* Activity Streams */ = {
+		E0E63D2D1D21DF3200DDF336 /* ActivityStreams */ = {
 			isa = PBXGroup;
 			children = (
-				E0E63D2E1D21E01A00DDF336 /* Activity.swift */,
```
Comment (agarwalashish) [inline MojioSDK.xcodeproj/project.pbxproj]: Why was ActivityLink.swift removed ?

**Pull request merged**